src/ophys_etl/transforms/trace_transforms.py

```diff
@@ -57,3 +63,109 @@ def extract_traces(movie_frames: np.ndarray, rois: List[coo_matrix],
                 traces[indx, time_slice] = raw_trace
 
     return traces
+
+
+def robust_std(x: np.ndarray) -> float:
+    """Compute the median absolute deviation assuming normally
+    distributed data. This is a robust statistic.
+
+    Parameters
+    ----------
+    x: np.ndarray
+        A numeric, 1d numpy array
+    Returns
+    -------
+    float:
+        A robust estimation of standard deviation.
+    Notes
+    -----
+    If `x` is an empty array or contains any NaNs, will return NaN.
+    """
+    mad = np.median(np.abs(x - np.median(x)))
+    return 1.4826*mad
+
+
+def noise_std(x: np.ndarray, filter_length: int = 31) -> float:
```

Comment:
this arg is never set aywhere. should it be getting an arg?

Reply:
Good catch, this was never specified other than the default in the original code.